deploy/jenkins-ci/Makefile

```diff
@@ -20,9 +20,9 @@ ifeq ($(RELEASE_TYPE),minor)
   # For minor, remove the -SNAPSHOT suffix from the Makefile
   BACKEND_VERSION_TO_RELEASE ?= $(shell semver bump release $(BACKEND_VERSION))
 endif
-ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
-  # For end-of-week snapshot release, substitute the -SNAPSHOT prefix in the Makefile with
-  # the value of $(RELEASE_TYPE) -- assuming value will be snapshot.{X}
+ifeq ($(findstring $(RELEASE_TYPE),snapshot edge),$(RELEASE_TYPE))
```

Comment:
Same comment here as in the ui PR. Looks like for snapshots this will expand to something like:
`ifeq ($(findstring snapshot.X,snapshot edge),snapshot.X)`
and because `snapshot.X` is not in `snapshot edge` it shouldn't evaluate to true.

I haven't tried the code, so I may be wrong :)

Reply:
You are correct here, good catch.

Reply:
Updated